test/strategy/auth0_test.exs

```diff
@@ -14,6 +14,11 @@ defmodule Ueberauth.Strategy.Auth0Test do
 
   @router SpecRouter.init([])
   @test_email "janedoe@example.com"
+  @session_options Plug.Session.init(
+                     store: Plug.Session.COOKIE,
+                     key: "_hello_key",
```

Comment:
Could this be `_my_key` so we keep the convention of Elixir examples (MyApp, MyPlug, etc) ?

Reply:
```suggestion
                     key: "_my_key",
```